### PR TITLE
Documentation changes for PowerScale 3.4.0 Release

### PR DESCRIPTION
**Pull request by meenakshidembi691**
# Description
Added attribution.md file

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, pep8, linting, or security issues
- [x] I have performed Ansible Sanity test using --docker default
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [x] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

### CODE REVIEW

**Comment by ShrinidhiRao15** [comment]
Please add the link to repository in the attribution file.

**Review by Bhavneet-Sharma — APPROVED** [review]

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -312,3 +312,10 @@ releases:
         name: writable_snapshots
         namespace: ''
     release_date: '2024-08-30'
+  3.4.0:
+    changes:
+      minor_changes:
+        - Added support for listing all the user in Info module.
```
Comment (sachin-apa) [inline changelogs/changelog.yaml:318]:
```suggestion
        - Added support to list all the paginated data for the users in Info module.
```

File: changelogs/changelog.yaml [hunk 2]
```diff
@@ -312,3 +312,10 @@ releases:
...
+        - Extended the functionality of ADS module by adding additional parameters.
```
Comment (sachin-apa) [inline changelogs/changelog.yaml:319]:
```suggestion
        - ADS module has been enhanced to support the additional parameters.
```

File: changelogs/changelog.yaml [hunk 3]
```diff
@@ -312,3 +312,10 @@ releases:
...
+        - Removed masked paths in response for NFS module.
```
Comment (sachin-apa) [inline changelogs/changelog.yaml:320]:
```suggestion
        - NFS module is enhanced to remove masked paths in response and also added declarative approach of managing clients.
```

File: docs/Release Notes.md [hunk 4]
```diff
@@ -80,12 +80,12 @@ New Features and Enhancements
...
 This section describes the features of the Ansible Modules for Dell PowerScale for this release.
 
-The Ansible Modules for Dell PowerScale release 3.3.0 supports the following features:
+The Ansible Modules for Dell PowerScale release 3.4.0 supports the following features:
 
-- Added support for create, modify, and delete Alert Channel.
-- Added support for create, modify, and delete Alert Rule.
-- Added support for create, and delete Writable Snapshots.
-- Added support for listing Writable Snapshots in Info module
+- Added support for listing all the user in Info module.
```
Comment (sachin-apa) [inline docs/Release Notes.md:85]: Please see the above comments and modify accordingly.

**Review by sachin-apa — APPROVED** [review]

**Review by Bhavneet-Sharma — APPROVED** [review]

**Review by felixs88 — APPROVED** [review]

**Review by trisha-dell — APPROVED** [review]

**Pull request merged**